test/DynamoCoreTests/DynamoCoreTests.csproj

```diff
@@ -72,6 +72,7 @@
       <Project>{a4794476-7d0e-41c0-ad83-4ab929c0a46c}</Project>
       <Name>ProtoScript</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\src\Libraries\CoreNodeModelsWpf\CoreNodeModelsWpf.csproj" />
```

Comment:
we should find a way to avoid adding this dependency, there is a strict boundary between the core/model binaries and the wpf/view binaries - so that Dynamo can be built and tested cross platform.

Reply:
Oops. Looks like I no longer needed that reference anyway. Thanks!